sphinx_js/renderers.py

```diff
@@ -309,6 +318,8 @@ def _template_vars(self, name, full_path, doclet):
         return dict(
             name=name,
             description=doclet.get('description', ''),
+            deprecated=doclet.get('deprecated', False),
```

Comment:
Just had an idea: someday we should factor out the common kwargs into a function we call from each of the function/class/attr renderers. Then we wouldn't have to repeat the tests 3x each time.

Reply:
Sphinx itself seems to use a set of mixins (https://github.com/sphinx-doc/sphinx/blob/master/sphinx/util/docfields.py), though I haven't looked closely enough to figure out how everything gets arranged there.